### PR TITLE
Add a todo to create ranking

### PR DESCRIPTION
**Pull request by mariannefabre**

### CODE REVIEW

File: src/components/Ranking.js [hunk 1]
```diff
@@ -15,6 +15,7 @@ const Ranking = () => {
     });
   }, []);
 
+  // TODO: DISPLAY SORTED OPTIONS
```
Comment (namirsab) [inline src/components/Ranking.js]: Hm, it seems you still have some work to do here. But nice, I see is also your first PR, keep up good work!

**Review by namirsab — COMMENTED** [review]
Nice first Pull Request, keep up good work

File: src/components/Ranking.js (hunk 1 above)
Comment (mariannefabre) [reply src/components/Ranking.js]: This is just a draft for now.. but thanks for reviewing it, will ping you when it's done

Comment (namirsab) [reply src/components/Ranking.js]: Ups, i didn't see it! Okay okay, my bad 😅

**Pull request merged**